.github/workflows/build.yaml

```diff
@@ -15,41 +15,67 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: 3.8
-
-      - name: Install sqlite
-        run: sudo apt-get install sqlite libyaml-dev
+          channels: conda-forge,defaults
+          channel-priority: strict
+          show-channel-urls: true
 
       - name: Update pip/wheel infrastructure
+        shell: bash -l {0}
         run: |
-          python -m pip install --upgrade pip
-          pip install wheel
+          conda install -y -q pip wheel
+
+      - name: Install sqlite
+        shell: bash -l {0}
+        run: |
+          conda install -y -q sqlite
 
       - name: Install postgresql (server)
         run: sudo apt install postgresql
 
       - name: Install postgresql Python packages
-        run: pip install psycopg2 testing.postgresql
+        shell: bash -l {0}
+        run: |
+          conda install -y -q psycopg2
+          pip install testing.postgresql
 
       - name: Install cryptography package for moto
-        run: pip install cryptography
+        shell: bash -l {0}
+        run: |
+          conda install -y -q cryptography
 
       - name: Install WebDAV packages for testing
-        run: pip install cheroot wsgidav
+        shell: bash -l {0}
+        run: |
+          pip install cheroot wsgidav
 
       - name: Install dependencies
+        shell: bash -l {0}
         run: |
           pip install -r requirements.txt
 
       # We have two cores so we can speed up the testing with xdist
       - name: Install pytest packages
-        run: pip install pytest pytest-flake8 pytest-xdist pytest-openfiles
+        shell: bash -l {0}
+        run: |
+          conda install -y -q \
+            flake8 \
+            pytest pytest-flake8 pytest-xdist pytest-openfiles
+
+      - name: List installed packages
+        shell: bash -l {0}
+        run: |
+          conda list
+          pip list -v
 
       - name: Build and install
-        run: pip install -v .
+        shell: bash -l {0}
+        run: |
+          pip install -v .
 
       - name: Run tests
-        run: pytest -r a -v -n 3 --open-files
+        shell: bash -l {0}
+        run: |
+          pytest -r a -v -n 3 --open-files
```

Comment:
Can you set the sqlalchemy environment variable here so that we have a chance to notice if a bunch of new warnings turn up?